packages/@adobe/spectrum-css-temp/components/steplist/index.css

```diff
@@ -13,255 +13,277 @@ governing permissions and limitations under the License.
 
-  /* Offset from the BOTTOM of the steplist */
-  --spectrum-steplist-label-labelOffset: 10px;
+  /* Distance between number bullet and label */
+  --spectrum-steplist-number-label-gap-small: 6px;
+  --spectrum-steplist-number-label-gap-medium: 8px;
+  --spectrum-steplist-number-label-gap-large: 12px;
+  --spectrum-steplist-number-label-gap-xlarge: 16px;
+
+  /* The distance between steps in vertical  */
+  --spectrum-steplist-vertical-separation: 28px;
 
-  /* Font size of the label */
-  --spectrum-steplist-label-text-size: 12px;
 }
 
 .spectrum-Steplist {
-  /* Contain child elements with positive z-index */
-  z-index: 0;
-  position: relative;
-
-  display: block;
-  vertical-align: top;
+  padding: 0;
   margin: 0;
-  padding-top: var(--spectrum-steplist-topPadding);
-  padding-left: var(--spectrum-steplist-sidePadding);
-  padding-right: var(--spectrum-steplist-sidePadding);
-
-  text-align: center;
-  white-space: nowrap;
-  font-size: 0; /* To remove html whitespace between inline elements */
-  line-height: 16px; /* in case the container changes it */
+  display: flex;
+  flex-wrap: nowrap;
+  overflow: hidden;
 }
 
-.spectrum-Steplist--interactive {
-  .spectrum-Steplist-label,
-  .spectrum-Steplist-markerContainer {
-    /* Show cursor on hit zone when clickable */
-    cursor: pointer;
-  }
+.spectrum-Steplist-marker-focus {
+  padding: var(--spectrum-steplist-marker-border-width);
+  border-width: var(--spectrum-steplist-marker-border-width);
+  border-color: transparent;
+  border-style: solid;
+  border-radius: 100%;
 }
 
-/* Small (w/o label or tooltip) */
-.spectrum-Steplist--small {
-  padding: var(--spectrum-steplist-small-topPadding) 0;
-
-  .spectrum-Steplist-item {
-    padding: 0 40px 0 0;
-    width: 80px;
-
-    /* Disable Label */
-    .spectrum-Steplist-label {
-      display: none;
-    }
-
-    &:first-child, &:last-child {
-      width: 20px;
-    }
-
-    /* Disable first line */
-    &:first-child {
-      .spectrum-Steplist-markerContainer {
-        margin-left: -10px;
-      }
-    }
+.spectrum-Steplist-item {
+  display: grid;
+  justify-items: center;
+  align-items: start;
+  grid-template-rows: auto;
+  grid-template-areas:
+    "chevron link";
+  .spectrum-Steplist-segment {
+    grid-area: chevron;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+  }
 
-    &:last-child {
-      .spectrum-Steplist-markerContainer {
-        margin-right: -10px;
-      }
+  .spectrum-Steplist-chevron {
+    &.is-reversed {
+      transform: scale(-1);
     }
   }
-}
-
-.spectrum-Steplist-item {
-  outline: none;
-  position: relative;
-  display: inline-block;
-  margin: 0;
-
-  /* Don't let tooltips mess up alignment */
-  vertical-align: bottom;
 
-  box-sizing: content-box;
-  width: var(--spectrum-steplist-step-width);
-  padding: 0 calc(var(--spectrum-steplist-step-width) / 2) 0 0;
+  /* Focus */
+  &:focus,
+  &.is-focused,
+  *:focus {
```

Comment:
It has been a while since I worked on this, but I'm pretty sure that I brought that over from spectrum-css directly without taking the time to investigate why it had so many different ways to select focused elements.

Reply:
Yeah, I'm pretty much assuming these are coming from spectrum-css, we have similar things all over the place that we're slowly cleaning up. Though if you're able to answer why there is a star selector, that'd be really helpful. Otherwise I'll investigate it at some point.

Reply:
i'll look over the css and clean it up a little 👍